salt/cloud/clouds/cloudstack.py

```diff
@@ -202,7 +202,11 @@ def get_project(conn, vm_):
     '''
     Return the project to use.
     '''
-    projects = conn.ex_list_projects()
+    try:
+        projects = conn.ex_list_projects()
+    except AttributeError:
+        # with versions <0.15 of libcloud this is causing an AttributeError.
```

Comment:
@brejoc, can you also add something like this here: `log.warning('Cannot get projects, you may need to update libcloud to 0.15 or later')`?